projects/smoke_ads1259/src/main.c

```diff
@@ -19,30 +27,29 @@ static void prv_rx_error_handler_cb(Ads1259StatusCode code, void *context) {
   }
 }
 
-static void prv_dump_queue(double *queue) {
-  LOG_DEBUG("\n\nDUMPING QUEUE....\n");
-  for (int reading = 0; reading < READING_QUEUE_LENGTH; reading++) {
-    LOG_DEBUG("VOLTAGE READING: %lf\n", queue[reading]);
-    queue[reading] = 0;
-  }
-  s_index = 0;
-}
-
 static void prv_periodic_read(SoftTimerId id, void *context) {
   double *queue = context;
   if (s_index < READING_QUEUE_LENGTH) {
+    LOG_DEBUG("=========READING # %i========= vref mv: %d\n", s_count++,
+              (int)(EXTERNAL_VREF_V * 1000));
     ads1259_get_conversion_data(&s_storage);
     queue[s_index] = s_storage.reading;
     s_index++;
     soft_timer_start_millis(CONVERSION_TIME_MS, prv_periodic_read, queue, NULL);
   } else {
-    prv_dump_queue(queue);
```

Comment:
Maybe it's moot since the smoke test has already served its purpose, but how does the smoke test user know what's been read now?

Reply:
It's logged a few lines above. It looks good in the logs, not so much in code